public/scripts/Table.js

```diff
@@ -45,15 +45,19 @@ var Table = React.createClass({
         });
     },
 
-    getSortedRows: function(cells) {
-       var sortedData = cells.sort(sort_by({item[colData.key]}: 'Quantity', asec));
-
+    getSortedRows: function() {
+      var cols = this.props.cols,
+          data = this.props.data;
+      return data.map(function(item) {
+          var cells = cols.map(function(colData) {
+          var sortedData = cells.sort(sort_by([colData.key]):'Quantity',asec);
```

Comment:
`sortBy` function is wrong..... 
look into [Creating, displaying, and sorting an array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) example......